icon-registry/icons-static/warning-triangle_x16.svg

```diff
@@ -0,0 +1,7 @@
+<svg viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@emilmilanov @emilyrohrbough The code for this warning trial icon seems oddly rather large for something that appears like a pretty simple icon. It might be worth seeing how this was constructed so it ends up being smaller (i.e. having a smaller footprint)